metaflow/plugins/kfp/kfp.py

```diff
@@ -194,8 +194,10 @@ def _command(
         def copy_log_cmd(log_file):
             cp_command = environment.get_boto3_copy_command(
                 s3_path=(
-                    f"{{datastore_root}}/{self.flow.name}/{{run_id}}/{step_name}"
-                    f"/${TASK_ID_ENV_NAME}/{log_file}"
+                    os.path.join(
+                        "{datastore_root}",
```

Comment:
Why is this not an `f string`? Maybe because you want to avoid the double brackets (`{{`)? But double brackets are used in the following line, so keeping it one f string will be consistent.

Reply:
Because that is replaced at runtime in https://github.com/zillow/metaflow/blob/f633d50b4e970ce85838e10d447cd1283ea3d17c/metaflow/plugins/kfp/kfp_step_function.py#L43-L47